### PR TITLE
Adding a few more tests

### LINKED ISSUE #1: Complete tests
Most of the utils.ts module has tests created but there is more to be done.

I need some more eyes on this and some help 😅

### PR DESCRIPTION
**Pull request by Chocrates**
Finally got off my butt and finished some tests.  
Closes #1

### CODE REVIEW

File: .npmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+@chocrates:registry=https://npm.pkg.github.com/
```
Comment (zkoppert) [inline .npmrc]: @Chocrates Looks like there is an [issue](https://github.com/ActionsDesk/invite-user/pull/3/checks?check_run_id=482878369#step:3:124) authenticating with the package registry.

Comment (zkoppert) [reply .npmrc]: I suspect that a GITHUB_TOKEN is needed to install the npm package from the registry given this [article](https://help.github.com/en/packages/publishing-and-managing-packages/about-github-packages) but I haven’t seen anything on *how* to do that.

Comment (Chocrates) [reply .npmrc]: Thanks Zack.  I know I had to generate a token to publish but Im surprised id need one to pull too.  Ill see what I can figure out

Comment (Chocrates) [reply .npmrc]: Yuck: https://github.community/t5/GitHub-API-Development-and/Download-from-Github-Package-Registry-without-authentication/m-p/35501/highlight/true#M3312

Comment (Chocrates) [reply .npmrc]: Ill give this a try this afternoon: https://github.community/t5/GitHub-Actions/Installing-npm-packages-from-the-GitHub-package-registry/m-p/46332/highlight/true#M6528

Comment (Chocrates) [reply .npmrc]: @zkoppert took me 23 tries but I got the build to work.  
Test is never going to work, see https://github.com/ActionsDesk/invite-user/pull/2
Not sure what to do there.

Comment (zkoppert) [reply .npmrc]: 🎉 Nice!

**Pull request merged**